spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/neo4j/Neo4jSpringJclLogging.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2012-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.boot.autoconfigure.neo4j;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.neo4j.driver.Logger;
+import org.neo4j.driver.Logging;
+
+/**
+ * Shim to use Spring JCL implementation, delegating all the hard work of deciding the
+ * underlying system to Spring and Spring Boot.
+ *
+ * @author Michael J. Simons
+ */
+final class Neo4jSpringJclLogging implements Logging {
```

Comment:
Shouldn't this be part of Spring Data Neo4j?

Reply:
I don't think that would help as this is in the part that just auto-configures the driver when SDN may not be around.